source/user-manual/reference/ossec-conf/global.rst

```diff
@@ -554,6 +557,24 @@ This setting toggles whether to query the external Wazuh Cyber Threat Intelligen
 | **Allowed values** | yes, no |
 +--------------------+---------+
 
+.. _reference_forward_to:
+
+forward_to
+^^^^^^^^^^
+
+Specifies the name of the socket where the output will be redirected. The socket must be defined previously.
```

Comment:
```suggestion
Specifies the :ref:`name of the socket <reference_ossec_socket_name>` where the output will be redirected. The socket must be defined previously.
```

Reply:
And to make this work, we would need to add `.. _reference_ossec_socket_name:` to https://github.com/wazuh/wazuh-documentation/blob/master/source/user-manual/reference/ossec-conf/socket.rst?plain=1#L28